wiki/Gameplay/Judgement/osu!/en.md

```diff
@@ -1,57 +1,83 @@
 # osu! judgement system
 
-## Hit result
-
-| Image | Name | Effect |
-| :-: | :-: | :-- |
-| ![](/wiki/shared/judgement/osu!/hit300.png "300") | 300 | The best achievable result for a hit object. Scoring nothing except 300s in a beatmap will award the player with the SS or SSH [grade](/wiki/Gameplay/Grade). Usually denoted with a blue colour. |
-| ![](/wiki/shared/judgement/osu!/hit300g.png "Geki") | 300 (激) Geki | A variant of the 300 result. Originates from [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan), called Elite Beat! in [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents). Granted upon hitting a 300 on the last element in a combo in which the player has scored only 300s. Getting a Geki will grant a sizable health bonus on top of the standard health increase for a 300. |
-| ![](/wiki/shared/judgement/osu!/hit300k.png "300 Katu") | 300 (喝) Katu or Katsu | A variant of the 300 result. Originates from [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan), called Beat! in [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents). Granted upon hitting a 300 on the last element in a combo in which the player has scored at least one 100, but no 50s or misses. Getting a Katu will give a small health bonus on top of the standard health increase for a 300. |
-| ![](/wiki/shared/judgement/osu!/hit100.png "100") | 100 | The second best achievable result for a hit object. Usually denoted with a green colour. |
-| ![](/wiki/shared/judgement/osu!/hit100k.png "100 Katu") | 100 (喝) Katu or Katsu | A variant of the 100 result. Originates from [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan), called Beat! in [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents). Granted upon hitting a 100 on the last element in a combo in which the player has scored at least one 100, but no 50s or misses. Getting a Katu will give a small health bonus on top of the standard health increase for a 100. |
-| ![](/wiki/shared/judgement/osu!/hit50.png "50") | 50 | The third best achievable result for a hit object, the last one before a miss. Usually denoted with an orange colour. Scoring a 50 will prevent the appearance of a Katu or a Geki at the combo's end. |
-| ![](/wiki/shared/judgement/osu!/hit0.png "Miss") | Miss | The worst possible result for a hit object. Scoring a Miss will reset the current combo to 0 and will prevent the appearance of a Katu or a Geki at the combo's end. |
+## Judgements
+
+A **judgement** (a.k.a. **hit result**) is the outcome of interacting with a [hit object](/wiki/Gameplay/Hit_object) during its hit window. Score and accuracy are calculated based on which judgements are received.
+
+These are commonly referred to by their score value (except for misses), i.e. a Great is usually called a "300" and so on.
```

Comment:
"a Great" looks weird, maybe play the interface label card here and make it "a `Great`" instead?

Reply:
you see, the lazer interface always says GREAT

should we just use caps for these?

Reply:
I'd say so -- for unification purposes

Reply:
kinda disagree with code block but optimally coloured text (blue for GREAT etc.) would be perfect